tensorflow/lite/micro/kernels/xtensa/conv.cc

```diff
@@ -31,6 +31,12 @@ limitations under the License.
 namespace tflite {
 namespace {
 
+struct OpData {
+  OpDataConv reference_op_data;
+
+  int scratch_tensor_index;
+};
```

Comment:
want to use this struct only for FUSION_F1? the scratch_tensor_index isn't really meaningful for hifimini or a reference fallback.

Reply:
#ifdef'd the extra member for F1 only.